akkaClient/src/main/scala/ca/pigscanfly/SwarmMessageClient.scala

```diff
@@ -0,0 +1,70 @@
+package ca.pigscanfly
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.{ContentTypes, HttpEntity, HttpMethods, HttpRequest, HttpResponse}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import ca.pigscanfly.httpClient.HttpClient
+import ca.pigscanfly.models.{Message, MessageDelivery, MessagePost, MessageRetrieval}
+import ca.pigscanfly.models.MessageRetrieval._
+import ca.pigscanfly.models.MessagePost.encoder
+import io.circe.parser
+import akka.http.scaladsl.model.headers.Cookie
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait SwarmMessageClient { this: HttpClient =>
+  implicit def actorSystem: ActorSystem
+
+  implicit def executionContext: ExecutionContext
+
+  def getMessages(url: String): Future[MessageRetrieval] = {
+    val cookieHeader = Cookie("JSESSIONID", "B120DCEBC05C9F6CE3FBCA259356C17E")
```

Comment:
I think this will have to come from the login endpoint rather than be hard coded yeah? (I know it's WIP so I assume that y'all are on it but just reading the code to make sure I understand).

Reply:
yes, its for testing purpose only. A direct call to get token can be made to swarm from Frontend.
When exposing our own endpoints to the above methods, we can pass token through header/cookie. which can be used here.
So this hardcoding would be removed.